frontend/app/services/TouchpointBackend.scala

```diff
@@ -107,10 +107,8 @@ object TouchpointBackend {
     )
   }
 
-  val Normal = TouchpointBackend(BackendType.Default)
-  val TestUser = TouchpointBackend(BackendType.Testing)
-
-  val All = Seq(Normal, TestUser)
+  lazy val Normal = TouchpointBackend(BackendType.Default)
```

Comment:
we do, but the health check checks that.  And if you're developing locally and only using UAT, do you really want to insist that DEV is valid?